sound/soc/sof/compress.c

```diff
@@ -147,8 +146,7 @@ static int sof_compr_free(struct snd_soc_component *component,
 	stream.comp_id = spcm->stream[cstream->direction].comp_id;
 
 	if (spcm->prepared[cstream->direction]) {
-		ret = sof_ipc_tx_message(sdev->ipc, &stream, sizeof(stream),
-					 &reply, sizeof(reply));
+		ret = sof_ipc_tx_message(sdev->ipc, &stream, sizeof(stream), NULL, 0);
```

Comment:
can we introduce an inline function, e.g. sof_ipc_tx_message_noreply() with fewer parameters.

Someone will screw this up one day otherwise....